projects/plugins/social/jetpack-social.php

```diff
@@ -86,5 +86,7 @@ function () {
 	return;
 }
 
+add_action( 'activated_plugin', array( 'Jetpack_Social', 'plugin_activation' ) );
```

Comment:
[register_activation_hook()](https://developer.wordpress.org/reference/functions/register_activation_hook/) could be used instead, which would not require a check for the correct file in the callback method.